### ISSUE TITLE
Version 1.1.8 not compatible with MC 1.21.0

### ISSUE THREAD

**Issue by winderine** [PotatoPresident/HeadIndex#22]
Running Minecraft 1.21.0 with Fabric Loader 0.16.9 and Fabric API 0.102.0 , running either [NoClip](https://modrinth.com/mod/noclip) 1.3.1 or [Head Index](https://modrinth.com/mod/headindex) 1.1.8 alone has no errors at all, but running both together causes a crash both on my PC and on my Aternos server. The first error thrown says that it's from sgui (part of Head Index) but the crash log says that NoClip caused the crash, so I am [posting this in both places](https://github.com/andantet/noclip/issues/16#issue-2717690442) to try to get further clarity. 

Full Log:
https://paste.ee/p/2wQ08

**Comment by dvitski** [comment]
HeadIndex contains SGUI for 1.21.2, while you are running Minecraft 1.21.

From my research, consider using HeadIndex 1.1.7.

**Comment by winderine** [comment]
> From my research, consider using HeadIndex 1.1.7.

Okay, thank you! That worked.

I hadn't actually tried logging into a world with Head Index alone, I assumed that loading the game without any errors meant it was running fine. Turns out version 1.1.8 fails to load in MC 1.21.0 with a rather terrifying "Invalid Player Data" error.

I've demonstrated here twice: 

https://paste.ee/p/dPN1q

For that reason I've changed the title of this bug post to "Version 1.1.8 not compatible with MC 1.21.0" which is just a labeling error on the mod distribution sites
